openai.finetunes/types.bal

```diff
@@ -115,7 +114,7 @@ public type OpenAIFile record {|
     string filename;
     string purpose;
     string status?;
-    record {}? status_details?;
+    record {}? status_details = ();
```

Comment:
did we check whether this works fine? (due to the same issue you were having in the other similar instances)

Reply:
Yes we tested